core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/workspacemanager/HttpWorkspaceManagerDAO.scala

```diff
@@ -162,7 +162,11 @@ class HttpWorkspaceManagerDAO(apiClientProvider: WorkspaceManagerApiClientProvid
   ): DataRepoSnapshotResource = {
     val snapshot = new DataRepoSnapshotAttributes().instanceName(instanceName).snapshot(snapshotId.toString)
     val commonFields =
-      new ReferenceResourceCommonFields().name(name.value).cloningInstructions(CloningInstructionsEnum.NOTHING)
+      new ReferenceResourceCommonFields()
+        .name(name.value)
+        // Note: that we're ignoring the passed in `cloningInstructions` is a known issue, addressed as part of
+        // https://github.com/broadinstitute/rawls/pull/2081
+        .cloningInstructions(CloningInstructionsEnum.NOTHING)
```

Comment:
Oh man but that PR is never gonna happen

Reply:
😆

Reply:
Fine by me, just wanted to save myself a headscratch the next time I wander into this area and squint at it.